custom-metrics-stackdriver-adapter/pkg/adapter/translator/query_builder.go

```diff
@@ -358,36 +425,43 @@ func joinFilters(filters ...string) string {
 	return strings.Join(nonEmpty, " AND ")
 }
 
+// Deprecated since FilterBuilder, use FilterBuilder instead
 func (t *Translator) filterForCluster() string {
 	projectFilter := fmt.Sprintf("resource.labels.project_id = %q", t.config.Project)
 	clusterFilter := fmt.Sprintf("resource.labels.cluster_name = %q", t.config.Cluster)
 	locationFilter := fmt.Sprintf("resource.labels.location = %q", t.config.Location)
 	return fmt.Sprintf("%s AND %s AND %s", projectFilter, clusterFilter, locationFilter)
 }
 
+// Deprecated since FilterBuilder, use FilterBuilder instead
 func (t *Translator) filterForMetric(metricName string) string {
 	return fmt.Sprintf("metric.type = %q", metricName)
 }
 
+// Deprecated since FilterBuilder, use FilterBuilder instead
 func (t *Translator) filterForAnyPod() string {
 	return "resource.type = \"k8s_pod\""
 }
 
+// Deprecated since FilterBuilder, use FilterBuilder instead
 func (t *Translator) filterForAnyNode() string {
 	return "resource.type = \"k8s_node\""
 }
 
+// Deprecated since FilterBuilder, use FilterBuilder instead
 func (t *Translator) filterForAnyContainer() string {
 	return "resource.type = \"k8s_container\""
 }
 
+// Deprecated since FilterBuilder, use FilterBuilder instead
```

Comment:
```suggestion
// Deprecated, use FilterBuilder instead
```
I think that's just as clear